kuma/wiki/jinja2/wiki/document.html

```diff
@@ -383,6 +383,12 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
       </div>
     </div> <!-- ends "main-content" -->
 
+    {% if waffle.switch('newsletter') and waffle.switch('newsletter_article') and waffle.flag('redesign') %}
```

Comment:
Does it matter that I do not have the `newsletter` and `newsletter_article` flags? I do have `redesign` enabled however.